components/org.wso2.carbon.identity.handler.event.account.lock/pom.xml

```diff
@@ -61,9 +61,24 @@
                         </Export-Package>
                         <Import-Package>
                             org.osgi.framework; version="${osgi.framework.imp.pkg.version.range}",
-                            org.osgi.service.component; version="${osgi.service.component.imp.pkg.version.range}"
+                            org.osgi.service.component; version="${osgi.service.component.imp.pkg.version.range}",
```

Comment:
Refer [1] in general. Use similar ordering and formatting.

[1] https://github.com/wso2-extensions/identity-governance/blob/master/components/org.wso2.carbon.identity.captcha/pom.xml